notebooks/datasets/nldas_time_series.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<p>There are easier ways to make a bbox from a polygon than writing geojson by hand in code. I'll find an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>